Modules/IO/MINC/src/itkMINCImageIO.cxx

```diff
@@ -456,6 +457,14 @@ void MINCImageIO::ReadImageInformation()
   dir_cos.Fill(0.0);
   dir_cos.SetIdentity();
 
+  //MINC stores direction cosines in RAS, need to convert to LPS for ITK
+  Matrix< double, 3,3 > RAS_tofrom_LPS;
+  RAS_tofrom_LPS.Fill(0.0);
```

Comment:
I think this operation is needless, as its result is annihilated by SetIdentity on the next line.